packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { inject, observer } from 'mobx-react'
+
+function storeMapper (stores) {
+  return {
+    subject: stores.classifierStore.subjects.current,
+  }
+}
+
+@inject(storeMapper)
```

Comment:
I think I'd prefer the subject viewer components to be pure functional component expecting a subject prop and have a container component that is connected to the MobX store. This way we can export the viewers independently and have them available for collections, talk, or separately for projects that might not want to use MobX, etc.

Reply:
Good idea